docs/development/testing/README.md

```diff
@@ -43,4 +43,21 @@ $ npm test
 
 ## Run all the tests for a specific demo
 
-// TODO
\ No newline at end of file
+// TODO
+
+## Record the tests on Cypress Dashboard
```

Comment:
Since changes have not been included on the workflow, imho It makes no sense to update the development guides.

Reply:
@carla-at-wiris ?

Reply:
It makes sense, then this would mean to close this PR since this is the only change made at code level

Reply:
No, I didn't mean that.